README.rst

```diff
@@ -0,0 +1,560 @@
+
+ # This parameter is useful when invoking the tool without specifying which
+ # images to synchronise. All the regions with glance servers registered in
+ # keystone are synchronised unless they are included in this parameter.
+ ignore_regions = Spain1
+
+ [experimental]
+
+ credential = user2,W91c2x5X2RpZF95b3VfdGhpbmtfdGhpc193YXNfdGhlX3JlYWxfcGFzc3dvcmQ/,http://server2:4730/v2.0,tenantid2
+ only_tenant_images = False
+ metadata_condition = image.is_public and image.user_properties.get('type', None) == 'baseimages'
+
+This configuration file defines two *targets*: ``master`` and ``experimental``. The first one
+synchronises all the public images with properties *nid* and/or *type* defined. The last one only
+synchronises images with ``type=baseimages``
+
+Security consideration
+----------------------
+
+Glancesyncs does not require *root* privileges. But at this version it requires
+read-only access to image directory ``/var/lib/glance/images`` (or making
+available a copy of all these files, or at least the subset that may be
+synchronised, in other path and then set the option *images_path*)
+
+It is strongly recommended:
+
+* creating an account to run glancesync only
+* creating a configuration file only readable by the glancesync account. This
+  is because the credentials should not be exposed to other users.
+
+Running
+=======
+
+Basic use
+---------
+
+Once installed all the dependencies, there is a way to run GlanceSync manually
+from the command line invoking the ``sync.py`` tool inside the GlanceSync
+distribution.
+
+When ``./sync.py`` is invoked without parameters, it synchronises the images from
+the master region to all the other regions with a glance endpoint registered in
+the keystone server (except the ones, if any, specified as a comma separated list
+in the ``ignore_regions`` parameter, inside the ``master`` section). The command
+can also receive as parameters the regions to synchronise.
+
+Advanced use
+------------
+
+By default, GlanceSync synchronises regions one by one. When the command line
+option *--parallel* is passed, GlanceSync synchronised several regions in
+parallel. The number or regions synchronised at the same time is determined by the
+parameter max_children in the main section. Default value is 1 (no parallel).
+When synchronisation runs on parallel, a directory with the pattern
+*sync_<year><month>_<hour><minute>* is created. Inside this, it is a file for each
+region with the log of the synchronisation process.
+
+The option *--dry-run* shows the changes needed to synchronise the images,
+but without doing the operations actually.
+
+Finally, the option *--show-status* is to obtain a report about the
+synchronisation status of the regions. A more detailed information of this is
+provided in the *Checking status* section.
+
+As pointed, GlanceSync can synchronised also from the master region to regions
```

Comment:
GlanceSync can synchronise

Caution!!! all the GlanceSync should be the same no Glancesync.